Tests/Svg.UnitTests/BoundsTests.cs

```diff
@@ -29,20 +29,20 @@ public void TestLineBounds()
         public void TestRectangleBounds()
```

Comment:
could you please also add your test svg with the respective png as usual as a regression test?